manifests/params.pp

```diff
@@ -11,9 +11,12 @@
   $disable_ssl               = false
   $open_ssl_listen_port      = undef
   $postgres_listen_addresses = 'localhost'
-
-  $database                  = 'postgres'
+  
+  $puppetdb_version          = $puppetdb::globals::version
+  $database                  = $puppetdb::globals::database
   $manage_dbserver           = true
+  $manage_pg_repo            = false
```

Comment:
I agree, if we want to be in a situation where the module "just works" for most of our users, then more often than not, we'll want to manage the pg_repos by default since most systems don't have 9.3/9.4 installed.